DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Centre/Administrator/CentreAdministratorsViewModel.cs

```diff
@@ -28,7 +28,8 @@ int page
             MatchingSearchResults = filteredItems.Count;
             SetTotalPages();
             var paginatedItems = GetItemsOnCurrentPage(filteredItems);
-            Admins = paginatedItems.Select(adminUser => new SearchableAdminViewModel(adminUser));
+            var returnPage = string.IsNullOrEmpty(searchString) ? 1 : page;

```

Comment:
Couple of things. 
Firstly, I feel like we want to ignore whitespace (though I made a comment on this in the JS). 
Secondly, you are going to want to double check a lot of these view models. Looks like the empty search case links to resetting the page to the first one, when we want that inverted. This is preventing the non-JS version from working as expected.